apps/web/src/lang/fr.json

```
@@ -10638,6 +10642,10 @@
     "defaultMessage": "1 à 5",
     "description": "Contract FTE range between one and five"
   },
+  "wYct8d": {
+    "defaultMessage": "Lorsque vous êtes admis dans un processus de recrutement de talents, vous acceptez de recevoir des notifications sur d’éventuelles possibilités qui répondent aux critères du processus. Ces notifications ne seront envoyées que si vous avez activé les notifications \"Alerte d’emploi\" dans la section des \"paramètres de notification\" de cette page. Vous pouvez aussi désactiver ces notifications pour chaque processus à l’aide des contrôles figurant sur les cartes fournies.",
```

replace quotation marks with guillemets
```suggestion
    "defaultMessage": "Lorsque vous êtes admis dans un processus de recrutement de talents, vous acceptez de recevoir des notifications sur d’éventuelles possibilités qui répondent aux critères du processus. Ces notifications ne seront envoyées que si vous avez activé les notifications « Alerte d’emploi » dans la section des « paramètres de notification » de cette page. Vous pouvez aussi désactiver ces notifications pour chaque processus à l’aide des contrôles figurant sur les cartes fournies.",
```

---

Fixed by [3a2852e](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971/commits/3a2852e43953fcf0fd7de74d65d4864c39a7e5eb)